test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,125 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class UtilsTest {
+
+	private static final int LIST_LENGTH = 10;
+	private Utils utils;
+	
+	@Before
+	public void setup() {
+		utils = new Utils();
+	}
+	
+	/**
+	 * Test isAnynull method
+	 */
+	
+	// Null input, return false
+	@Test
+	public void emptyInput_returnsIncorrect() {
+		ArrayList listToTest = generateEmptyList();
+		Object[] arrayToTest = new Object[listToTest.size()];
+		arrayToTest = listToTest.toArray();
+		assertEquals(false, utils.isAnyNull(arrayToTest));
+	}
+	
+	// List with null, return true
+	@Test
+	public void withNullInput_returnsCorrect() {
+		ArrayList<Object> listToTest = generateListWithNull();
+		Object[] arrayToTest = new Object[listToTest.size()];
+		arrayToTest = listToTest.toArray();
+		assertEquals(true, utils.isAnyNull(arrayToTest));
```

Comment:
Similar to above, do take a look at assertTrue and try using it instead of using assertEquals in the case of booleans.